### PR TITLE
apply time step correction

### PR DESCRIPTION
**Pull request by aleeciu**
Applied time step correction, after #235

### CODE REVIEW

File: climada/hazard/tc_rainfield.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def rainfield_from_track(track, centroids, dist_degree=3, intensity=0.1):
             fradius_km[pos] = np.sqrt(dd) * 111.12
 
             rainsum += _RCLIPER(track.max_sustained_wind.values[node],
-                                inreach, fradius_km)
+                                inreach, fradius_km)*track.time_step.values[node]
```
Comment (tovogt) [inline climada/hazard/tc_rainfield.py:181]: Linting:
```suggestion
                                inreach, fradius_km) * track.time_step.values[node]
```

**Comment by emanuel-schmid** [comment]
Looks good. It would be awesome if we had it covered in a unit test. 😉

**Comment by aleeciu** [comment]
> Looks good. It would be awesome if we had it covered in a unit test. 😉

Will do. So overall rainfall computed based on different timesteps can still differ by a bit. For randomly picked track for example, one could get 276259.044 mm or 276893.354 mm with 1h or 0.5h time step, respectively. What is the best way to assert such large absolute dfference but small relative difference in the unit test? assertAlmostEqual will fail I guess..

**Comment by tovogt** [comment]
To test that the relative distance between the two is less than 1%, you could use something like this: 
```
np.testing.assert_allclose(276259.044, 276893.354, rtol=1e-2)
```

**Comment by aleeciu** [comment]
> Looks good. It would be awesome if we had it covered in a unit test. 😉

done

**Comment by aleeciu** [comment]
> To test that the relative distance between the two is less than 1%, you could use something like this:
> 
> ```
> np.testing.assert_allclose(276259.044, 276893.354, rtol=1e-2)
> ```

thanks

**Review by tovogt — APPROVED** [review]

**Pull request merged**